website/docs/docs/cloud/use-visual-editor.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Edit and create dbt modelsr" 
+id: use-visual-editor      
+sidebar_label: "Edit and create dbt models" 
+description: "Access and use the visual editor to create or edit dbt models through a visual, drag-and-drop experience inside of dbt Cloud." 
+---
+
+# Edit and create dbt models <Lifecycle status='beta'/> 
+
+<p style={{ color: '#717d7d', fontSize: '1.1em' }}>
+Access and use the dbt Cloud visual editor to create or edit dbt models through a visual, drag-and-drop experience. Use the built-in AI for custom code generation in your development experience.
+</p>
+
+:::tip Beta feature
+The visual editor provides users with a seamless and drag-and-drop experience inside of dbt Cloud. It's available in private beta for [dbt Cloud Enterprise accounts](https://www.getdbt.com/pricing). 
+
+To join the private beta, [register your interest](https://docs.google.com/forms/d/e/1FAIpQLScPjRGyrtgfmdY919Pf3kgqI5E95xxPXz-8JoVruw-L9jVtxg/viewform) or reach out to your account team to begin this process.
+:::
+
+## Prerequisites
+- You have a [dbt Cloud Enterprise](https://www.getdbt.com/pricing) account.
+- You have a [developer license](/docs/cloud/manage-access/seats-and-users) ??? WHO can access editor? dev license? read-only?
+- You have a dbt Cloud project created already with dev credentials set up.
+- Successful job run on Production or Staging [environment](/docs/dbt-cloud-environments)
+- Have AI-powered features toggle enabled
+
+## Access visual editor
+
+Before accessing the editor, you should have a dbt Cloud project already set up. This includes a Git repository, data platform connection, environments, and developer credentials. If you don't have this set up, please contact your dbt Cloud Admin.
+
+To access the visual editor:
+- Type in the following URL, replacing the ACCOUNT_ID and ENVIRONMENT_ID with your own account and environment ID: `https://ACCESS_URL/visual-editor/ACCOUNT_ID/env/ENVIRONMENT_ID/`
```

Comment:
is this the current way to access the editor right now (before it goes under the dev tab)?

Reply:
keep as it rn